docs/kyma/docs/04-02-local-installation-from-release.md

```diff
@@ -94,8 +94,9 @@ To install Kyma, follow these steps:
   ./scripts/is-installed.sh
   ```
 
-Read the **Reinstall Kyma** document to learn how to reinstall Kyma without deleting the cluster from Minikube.
-To learn how to test Kyma, see the **Testing Kyma** document.
+Read [this](#installation-reinstall-kyma) document to learn how to reinstall Kyma without deleting the cluster from Minikube.
+
+To learn how to test Kyma, see [this](#installation-reinstall-kyma) document.
```

Comment:
Are you sure it's the right link? It leads to `Reinstall Kyma`, not to any testing scenario.